frontend/src/types/analytics.ts

```diff
@@ -47,7 +47,7 @@ export type Events = {
    *     when not paired with a `GET_MEDIA` event?
    */
   RIGHT_CLICK_IMAGE: {
-    identifier: string
+    id: string
```

Comment:
I know this isn't something introduced in this PR and that this change is to the match the other events, but it would be good to keep in mind the potential ambiguity here introduced by the fact that `id` and `identifier` are very different for results on the API, ingestion server, and Elasticsearch side. Especially considering that analysis on these events would likely intersect with making queries against Elasticsearch, the API database, or the catalog database, all of which use `identifier` for the information stored in this payload property.

Reply:
>  keep in mind the potential ambiguity here introduced by the fact that id and identifier are very different for results on the API
I'll open a discussion because this is really an important issue.

For the API _results_ and the frontend, it's simple - there's only the `id` field in the frontend that comes from the API responses. The API uses Openverse UUID that is created by the catalog as the response's `id` field (not the Elasticsearch `id`). One place where the `identifier` property is used is in the homepage images json - which I'll open an issue to rename with `id`.

I tried to find how the `identifier` field becomes the API response's `id` field - and couldn't 😮